hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/ContainerEndpoint.java

```diff
@@ -141,9 +140,12 @@ public Response getContainers(
       // Send back an empty response
       return Response.status(Response.Status.NOT_ACCEPTABLE).build();
     }
+
     long containersCount;
-    Collection<ContainerMetadata> containerMetaDataList =
-        containerManager.getContainers(ContainerID.valueOf(prevKey), limit)
+    List<ContainerMetadata> containerMetaDataList =
+        // Get the containers starting from the prevKey+1 which will skip the
+        // container having prevKey ID
+        containerManager.getContainers(ContainerID.valueOf(prevKey + 1), limit)
```

Comment:
one small question: this behaviour change won't have effect in the UI? or other places this method is used? 
or it was working incorrectly before so it will work correctly now?

Reply:
No, currently we have no UI components that are utilizing this specific endpoints alone `http://localhost:9888/api/v1/containers/` but we do have UI components that utilize this UI path such as 
`http://localhost:9888/api/v1/containers/unhealthy` but the changes that we have made does not impact them.

Reply:
okay, thanks for checking it @ArafatKhan2198!